### PR TITLE
PD CSI Driver presubmit windows e2e tests do the same as linux

### PR DESCRIPTION
**Pull request by mauriciopoppe**
I'll update these presubmit tests first and then the periodic tests

/cc @jingxu97 @mattcary

### CODE REVIEW

File: config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml [hunk 1]
```diff
@@ -205,34 +205,35 @@ presubmits:
...
+      resources:
+        requests:
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
+          # during the tests more like 3-20m is used
+          cpu: 2000m
     annotations:
       testgrid-dashboards: provider-gcp-compute-persistent-disk-csi-driver
       testgrid-tab-name: win2019-presubmit-gcp-compute-persistent-disk-csi-driver
```
Comment (jingxu97) [inline config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml:234]: you want to modify to windows-2019 here too?

Comment (mauriciopoppe) [reply config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml:234]: yes, both 2019 and 20H2, we usually don't run these presubmit tests that often anyways, the important ones will be the CI tests that I'll update later

Comment (jingxu97) [reply config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml:234]: oh, I just mean to rename to "windows-2019-presubmit..." for consistency

Comment (mauriciopoppe) [reply config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml]: oh ok, sure, changed to windows-2019

**Pull request merged**